### PR TITLE
Example scenarios for SAP

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/apps/sap-production.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+* Mission-critical workloads running on SAP
+* Non-critical SAP workloads 
+* Test environments for SAP that simulate a high-availability environment.
+
+## Architecture
+
+![Architecture overview of a production SAP environment in Azure][architecture]
+
+This example includes a high availability configuration for an Oracle database, SAP central services, and multiple SAP application servers running on different virtual machines. The Azure network uses a [hub-and-spoke topology][/azure/architecture/reference-architectures/hybrid-networking/hub-spoke] for security purposes. The data flows through the solution as follows:
```
Comment (MikeWasson) [inline docs/example-scenario/apps/sap-production.md]: Link is mis-formatted

File: docs/example-scenario/apps/sap-production.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Users access the SAP system via the SAP user interface, a web browser, or other client tools like Microsoft Excel. An ExpressRoute connection provides access from the organization's physical location to resources running in Azure.
```
Comment (MikeWasson) [inline docs/example-scenario/apps/sap-production.md]: physical location > on-premises network?

File: docs/example-scenario/apps/sap-production.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/example-scenario/apps/sap-production.md]: Fixed.

File: docs/example-scenario/apps/sap-production.md (hunk 2 above)
Comment (alexbuckgit) [reply docs/example-scenario/apps/sap-production.md]: Fixed.

File: docs/example-scenario/apps/sap-production.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+## Considerations
+
+Recommended practices are defined for building highly available SAP environments in Azure. For more information, see [High-availability architecture and scenarios for SAP NetWeaver](/azure/virtual-machines/workloads/sap/sap-high-availability-architecture-scenarios).
+For additional information, see [High availability of SAP applications on Azure VMs](/azure/virtual-machines/workloads/sap/high-availability-guide).
+* Oracle databases also have recommended practices for Azure. For more information, see [Designing and implementing an Oracle database in Azure](/azure/virtual-machines/workloads/oracle/oracle-design). 
+*Oracle Data Guard is used to eliminate single points of failure for mission critical Oracle databases. For more information, see [Implementing Oracle Data Guard on a Linux virtual machine in Azure](/azure/virtual-machines/workloads/oracle/configure-oracle-dataguard).
```
Comment (MikeWasson) [inline docs/example-scenario/apps/sap-production.md]: List formatting error

File: docs/example-scenario/apps/sap-production.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+|Small|30000|DS13_v2|4xP20, 1xP20|DS11_v2|1x P10|DS13_v2|1x P10|[Small](https://azure.com/e/45880ba0bfdf47d497851a7cf2650c7c)|
+|Medium|70000|DS14_v2|6xP20, 1xP20|DS11_v2|1x P10|4x DS13_v2|1x P10|[Medium](https://azure.com/e/9a523f79591347ca9a48c3aaa1406f8a)|
+Large|180000|E32s_v3|5xP30, 1xP20|DS11_v2|1x P10|6x DS14_v2|1x P10|[Large](https://azure.com/e/f70fccf571e948c4b37d4fecc07cbf42)|
+Extra Large|250000|M64s|6xP30, 1xP30|DS11_v2|1x P10|10x DS14_v2|1x P10|[Extra Large](https://azure.com/e/58c636922cf94faf9650f583ff35e97b)|
+
+Note: pricing is a guide and only indicates the VMs and storage costs. It excludes networking, backup storage, and data ingress/egress charges.
+
+* [Small](https://azure.com/e/45880ba0bfdf47d497851a7cf2650c7c): A small system consists of VM type DS13_v2 for the db server with 8x vCPUs, 56GB RAM and 112GB temp storage, additionally five 512GB premium storage disks. An SAP Central Instance servers using a DS11_v2 VM types with 2x vCPUs 14GB RAM and 28GB temp storage. A single VM type DS13_v2 for the SAP application server with 8x vCPUs, 56GB RAM and 400GB temp storage, additionally one 128GB premium storage disk.
```
Comment (MikeWasson) [inline docs/example-scenario/apps/sap-production.md]: "db" > DB or database

Comment (MikeWasson) [reply docs/example-scenario/apps/sap-production.md]: servers > server

Comment (MikeWasson) [reply docs/example-scenario/apps/sap-production.md]: Might be more readable to break these bullet items into sublists? Or it there any way to put this info in the table?

File: docs/example-scenario/apps/sap-production.md (hunk 3 above)
Comment (alexbuckgit) [reply docs/example-scenario/apps/sap-production.md]: Fixed.

**Pull request merged**